client/providers/OmniChannelProvider.tsx

```diff
@@ -96,8 +96,9 @@ const OmnichannelEnabledProvider: FC = ({ children }) => {
 		setContextValue((context) => ({
 			...context,
 			agentAvailable: user?.statusLivechat === 'available',
+			routeConfig,
```

Comment:
we created 3 different providers to inform only the necessary and typed information for each case, so you practically ignore all this logic